src/multiple_choice/config.md

```diff
@@ -0,0 +1,14 @@
+## Config Options
+
+- `answerColoring`
+    - `DEFAULT_COLORING` (default): If a row was ticked or left unticked correctly it is shown green and red otherwise.
+    - `ALTERNATE_COLORING`: Only color falsely ticked rows red and rows that should have been ticked green.
+- `colorQuestionTable`
+    - `true`: Color the table with your chosen answers according to their correctness.
+    - `false` (default): Don't modify the table with your chosen answers.
+- `colorAnswerTable`
+    - `true` (default): Color the table with the correct answers according to your chosen answers.
+    - `false`: Don't modify the table with the correct answers.
+- `hideAnswerTable`
+    - `true`: Only show the table with your chosen answers.
+    - `false` (default): Show the table with the correct answers after confirming your chosen answers.
```

Comment:
- Include type.
- Use bool instead of string.
```suggestion
- `colorQuestionTable` (bool)
    - `True`: Color the table with your chosen answers according to their correctness.
    - `False` (default): Don't modify the table with your chosen answers.
- `colorAnswerTable` (bool)
    - `True` (default): Color the table with the correct answers according to your chosen answers.
    - `False`: Don't modify the table with the correct answers.
- `hideAnswerTable` (bool)
    - `True`: Only show the table with your chosen answers.
    - `False` (default): Show the table with the correct answers after confirming your chosen answers.
```